mindjunkies/forums/forms.py

```
@@ -0,0 +1,21 @@
+
+    def save(self, commit=True):
+        instance = super(ForumTopicForm, self).save(commit=False)
+        instance.slug = slugify(instance.title)
+        if commit:
+            instance.save()
+        return instance
+
+class ForumReplyForm(forms.ModelForm):
+    class Meta:
+        model=ForumReply
+        fields=['content']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)